packages/data-grid/src/Grid.tsx

```diff
@@ -421,9 +421,19 @@ export const Grid = function Grid<T>(props: GridProps<T>) {
       if (s) {
         s.scrollLeft += deltaX;
         s.scrollTop += deltaY;
+
+        if (
+          !(
+            (
+              s.clientHeight - s.scrollHeight - s.scrollTop <= 0 || // reached the bottom
```

Comment:
scrollTop was a decimal for me and caused the old logic to break. This would avoid that

Reply:
logic not working for me and @origami-z , perhaps it would work if we rounded it?